spec/integration_spec.rb

```diff
@@ -81,7 +81,7 @@
 
     it { expect(response.except(:result)).to eq response_type: :unstructured,
                                                 test_results: [],
-                                                status: :passed_with_warnings,
+                                                status: :failed,
```

Comment:
**(1)** I had to modify this because of the change made in https://github.com/mumuki/mumuki-laboratory/issues/1054

Reply:
:+1: